src/Address.svelte

```diff
@@ -95,7 +95,15 @@
   {/if}
   <div class="wrapper">
     {#if addressType === AddressType.Org}
-      <a use:link href={`/${nameOrAddress}`}>{addressLabel}</a>
+      <!-- svelte-ignore a11y-invalid-attribute -->
+      <a
+        use:router.link={{
```

Comment:
This way of defining the use action seems weird.. I would prefer to keep it as `use:link`